fetch_resources.sh

```diff
@@ -26,12 +26,15 @@ sed -i "2i export PYTHON_VERSION=$PYTHON_VERSION" run_tempest.sh
 sed -i "/PYTHON_CMD=/c PYTHON_CMD=\"$TEMPEST_VENV_DIR/bin/python$PYTHON_VERSION\"" sanity-run.sh
 sed -i "/PYTHON_CMD=/c PYTHON_CMD=\"$TEMPEST_VENV_DIR/bin/python$PYTHON_VERSION\"" master-run.sh
 
+cd /root
+eval "$(<env.sh)"
+export cli_pod=$(kubectl -n openstack get pod -l application=keystone,component=client -ojsonpath='{.items[*].metadata.name}')
```

Comment:
Please confirm parameter here. You gave "-ojsonpath". I think you missed to give space between  "-o" and "jsonpath".

Reply:
It works both with space and without space